### ISSUE TITLE
how to compile and run it on linux?

### ISSUE THREAD

**Issue by yukulele** [jwdj/EasyABC#5]

**Comment by mmehl** [comment]
see "[using_EasyABC_in_Linux.txt](https://github.com/jwdj/EasyABC/blob/master/using_EasyABC_in_Linux.txt)"

Can this issue be closed?

**Comment by yukulele** [comment]
I can run it, thanks!

some remarks:
- I think a minimum of information should be added in the readme with a link to the installation files.
- The txt format is not easy to read, an md format would be less intimidating.

Thank you for your help!

**Comment by yukulele** [comment]
I can run it, thanks!

some remarks:
- I think a minimum of information should be added in the readme with a link to the installation files.
- The txt format is not easy to read, an md format would be less intimidating.

Thank you for your help!

**Comment by mausch** [comment]
For anyone looking to run EasyABC on Linux, give this a try: https://github.com/jwdj/EasyABC/issues/72

**Comment by mtarenskeen** [comment]
>     * The txt format is not easy to read, a markdown format would be less intimidating.

?? What's intimidating about the simplest text format? It's just text. Nothing more, nothing less.
I personally find *.md files more intimidating when I need to write and read/view them on my Linux box.

**Comment by aupfred** [comment]
@jwdj I think this can be closed. It is running on various Linux version with version of python up to 3.13.1.

**Comment by mtarenskeen** [comment]
For the first time since quite some time at least it is showing up and running now on my system (Fedora 41, Python 3.13.1).
But I am still having a lot of issues. Maybe I should just report these separately? They may be unrelated to the issues that are fixed here as a result of this discussion.

**Comment by revad** [comment]
Yes. Open issues yourself then you can close them when fixed or superseded.
 As @topchyan did when he raised https://github.com/jwdj/EasyABC/issues/78
which  is what got discussed and fixed.

@aupfred and I are targetting python 3.13.
